### ISSUE TITLE
Feature Request: Remove "Priority" from Being Visible in the DD User Portal

### ISSUE THREAD

**Issue by DavidSolutionsIT** [DeskDirector/UserVoice#536]
![PriorityDDUserPortal](https://user-images.githubusercontent.com/79667593/109216168-caacc300-7779-11eb-98cf-27174f02cc7f.png)

[PriorityDDUserPortal.pdf](https://github.com/DeskDirector/UserVoice/files/6046064/PriorityDDUserPortal.pdf)

Our clients don’t need to see it; we don’t really care for them to see it; we don’t want them to see it. That’s the biggest reason.

Our customers don’t understand what the priority means, they (therefore) interject their own definition of what P3 vs. P4 means (not realizing we don’t strictly use P1-P4, but rather work off of an Impact/Urgency matrix that has nine levels, not 4), and therefore it causes confusion, consternation, and (unnecessary) questions/communication.

In short: we spend more time explaining priority than getting to work on the ticket and educating the user base on priority (as a classification) that we don't really use is not something we desire to do. So, basically, it’s an undesirable feature; something we would like to turn off.

**Comment by Nness** [comment]
Could you log this through our support portal?

**Comment by DavidSolutionsIT** [comment]
You're going to have to clarify what you mean, because I already had a support ticket open.

See attached.

David Wall
SERVICE DESK TEAM LEADER
[cid:tslogo1536restransparent-resized_f3f53cc9-9ad1-4054-91d9-e948bd3f54f7.png]
    P. 1-204-325-1000
    E. service@solutionsit.ca<mailto:hservice@solutionsit.ca>
    www.solutionsit.ca<http://www.solutionsit.ca/>

To reach our service desk, please call 1 866 325 7536

**Comment by Nness** [comment]
Thanks for the image, I have updated the link to include the ticket ID.

**Comment by DavidSolutionsIT** [comment]
Ah. You needed that ticket number. Gotcha. Yeah, I blacked that out. My bad.

Thanks again!

**Comment by iskolares** [comment]
Related ticket #26498

**Comment by Aycorn** [comment]
Related ticket #515090

"
Priority is for us, its not for clients. I would hide priority from view on tickets for all portal users.

The reasoning is that priority is for us working the tickets it is not for the end user. All our tickets come in under “Medium” priority simply because we don’t want clients seeing any of their issues marked as ‘Low’ priority. This forces us to escalate some tickets to high priority when they should not be, they should be escalated from low to medium.
"

**Comment by Jarrod-RDS** [comment]
Over a year old, not sure how hopeful I should be
We have clients seeing Priority - Medium, and contact us because they feel their issue is high priority. In reality their issue is going to be fixed in the same amount of time.
They don't need to see priority, it just causes issues like these.
